lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -15,6 +15,18 @@
  * @access private
  */
 class WP_Theme_JSON_6_1 extends WP_Theme_JSON_6_0 {
+	const ELEMENT_BUTTON_CLASS_NAME = 'wp-element-button';
```

Comment:
So this is expected to grow - at least temporarily?

Let's make experimental as it's `public`.

```suggestion
	const EXPERIMENTAL_ELEMENT_BUTTON_CLASS_NAME = 'wp-element-button';
```

Reply:
or even 

```suggestion
	const __experimental_ELEMENT_BUTTON_CLASS_NAME = 'wp-element-button';
```

to make it come up in the code search